packages/card-support/src/components/DownloadClientData/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -69,10 +69,6 @@ Object {
           title="All Locations"
           value=""
         />
-        <TreeNode
-          title="CSB Hopital Bouficha"
-          value="e2b4a441-21b5-4d03-816b-09d45b17cad7"
-        />
```

Comment:
Q: why did this change

Reply:
That's the correct behavior, no location selected at render

Reply:
Correct me if am wrong, looks like this is not a snapshot of the options `after a select`, but a general snapshot of the options that exist in this particular treeSelect, during first render there should be 2 options the `All locations` and `root location`, as shown [here](https://github.com/opensrp/web/blob/e011b31af03680ef4ed3331c5e32d45902ddef8b/packages/card-support/src/components/DownloadClientData/index.tsx#L179)

Reply:
![Screenshot from 2022-01-27 15-53-16](https://user-images.githubusercontent.com/10197807/151362773-07e00d7c-8cc0-4248-9d79-c89a88a79bd8.png)
Only one label until you click the dropdown

Reply:
I take it back, the tests failed. I think it's a disconnect between the CI and local setup. Ought to be checked out